File: test/test06.py
#!/usr/bin/python2.7 -u
import re
var = "hello\n"
var = var.rstrip()
array = var.split('l')
print array
line = '-'.join(array)
print line
line = re.sub(r'(?i)l', 's', line)
print line
re.search(r'(?i)l', line)
print line
re.search(r'p', line)
print line

File: test/test06.pl
#!/usr/bin/perl -w
$var = "hello\n";
chomp $var;
@array = split ('l', $var);
print "@array\n";
$line = join ('-', @array);
print "$line\n";
$line =~ s/l/s/gi;
print "$line\n";
$line =~ /l/i;
print "$line\n";
$line =~ /p/;
print "$line\n";